library-docker/bin/restart-kiwix-serve.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+export PATH=/usr/local/bin:/usr/sbin:$PATH
+
+start () {
+  stop  
+  echo "Starting kiwix-serve..."
+  kiwix-serve --daemon --port=8000 --library --threads=16 --verbose --nodatealias library.kiwix.org.xml
+}
+
+stop () {
```

Comment:
The way how I did it earlier was more secure. There is an assuption here that there is no other `kiwix-serve` running on the whole system. This is the case most of the time, but time to time we start test `kiwix-serve` server on the same system, this would kill it.

Reply:
OK this wasn't documented. What's the use case for this other kiwix-serve? If it's for debugging (the library mostly), I'd advise you start a kiwix-serve container with same mount points instead as this would be cleaner and would have the added benefit of being able to expose its HTTP port.
Feels odd to handle this special debug case here

Reply:
It is a debug case. But having a docker killing process outside Docker is odd as well. Maybe we should but the pid in `/var/run/kiwix-serve.pid` so things are clean?